core/workspace_drag_surface_svg.ts

```diff
@@ -165,23 +159,19 @@ export class WorkspaceDragSurfaceSvg {
   setContentsAndShow(
       blockCanvas: SVGElement, bubbleCanvas: SVGElement,
       previousSibling: Element, width: number, height: number, scale: number) {
-    if (this.SVG_.childNodes.length) {
+    if (this.SVG.childNodes.length) {
       throw Error('Already dragging a block.');
     }
-    this.previousSibling_ = previousSibling;
+    this.previousSibling = previousSibling;
     // Make sure the blocks and bubble canvas are scaled appropriately.
     blockCanvas.setAttribute(
         'transform', 'translate(0, 0) scale(' + scale + ')');
     bubbleCanvas.setAttribute(
         'transform', 'translate(0, 0) scale(' + scale + ')');
-    // AnyDuringMigration because:  Argument of type 'number' is not assignable
-    // to parameter of type 'string'.
-    this.SVG_.setAttribute('width', width as AnyDuringMigration);
-    // AnyDuringMigration because:  Argument of type 'number' is not assignable
-    // to parameter of type 'string'.
-    this.SVG_.setAttribute('height', height as AnyDuringMigration);
-    this.SVG_.appendChild(blockCanvas);
-    this.SVG_.appendChild(bubbleCanvas);
-    this.SVG_.style.display = 'block';
+    this.SVG.setAttribute('width', String(width));
+    this.SVG.setAttribute('height', String(height));
```

Comment:
The style guide doesn't have an opinion on this, but I prefer template literals since they take up less characters.

Reply:
Ah nice, I hadn't thought about that. There are other places using String()/toString(), so let's fix that up separately.